tools/tf2ngraph.py

```diff
@@ -94,6 +97,17 @@ def dump_json(json_name, optional_params=None, shape_hints=None):
             json.dump(dict_to_dump, fp)
 
 
+# This function controls how errors are handled.
+# For developers/debugging set raise_exception to True
+def exit_on_error(success, error_message, raise_exception=False):
```

Comment:
the term raise_exception is a bit misleading as we are not really raising exception. Rename it to use_assert or something like assert_on_failure

Reply:
Thinking more on this, why do we event need this flag? If we really want to assert (which shouldn't be done in a  user focused script) - then we need to modify all the call sites with that change. So we should just remove this IMO.